### PR TITLE
global: drop support for Python<3.5

### PR DESCRIPTION
**Pull request by adrien-berchet**

### CODE REVIEW

**Comment by adrien-berchet** [comment]
@jirikuncar here is the specific PR to change the supported Python versions

File: setup.py [hunk 1]
```diff
@@ -86,13 +85,11 @@
...
     tests_require=tests_require,
     classifiers=[
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
```
Comment (jirikuncar) [inline setup.py:92]:
```suggestion
        'Programming Language :: Python :: 3.8',
        'Programming Language :: Python :: 3.9',
```

File: tox.ini [hunk 2]
```diff
@@ -8,7 +8,7 @@
 # details.
 
 [tox]
-envlist = py27, py34, py35, py36
+envlist = py36, py37, py38
```
Comment (jirikuncar) [inline tox.ini:11]:
```suggestion
envlist = py36, py37, py38, py39
```

**Review by jirikuncar — COMMENTED** [review]
Can you please add 3.9 to the list https://github.com/inveniosoftware/dictdiffer/blob/master/.github/workflows/tests.yml#L31 ?

**Comment by adrien-berchet** [comment]
> Can you please add 3.9 to the list https://github.com/inveniosoftware/dictdiffer/blob/master/.github/workflows/tests.yml#L31 ?

Sure, done.

**Comment by adrien-berchet** [comment]
@jirikuncar could you run the CI please? :)

**Review by jirikuncar — APPROVED** [review]

**Pull request merged**